examples/raster_management.py

```diff
@@ -1,15 +1,27 @@
 #!/usr/bin/env python3
 
+from pprint import pprint
 from picterra import APIClient
 
 # Set the PICTERRA_API_KEY environment variable to define your API key
 client = APIClient()
+# The Id of a folder/project you own
+folder_id = "abcdefghijklmnopwrstuvz"
```

Comment:
Let's show a proper uuid here, just to server as a good example of what the id should look like

Reply:
Checked